Program.cs

```diff
@@ -1,52 +1,62 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace Task1
+namespace Task2
 {
     class Program
     {
-        static bool Validation(out double a, out double b, out double c)
+        static bool Validation(out int a)
         {
-            Console.WriteLine("Enter the values of the sides of the triangle");
             try
             {
-                a = double.Parse(Console.ReadLine());
-                b = double.Parse(Console.ReadLine());
-                c = double.Parse(Console.ReadLine());
+                a = int.Parse(Console.ReadLine());
             }
             catch
             {
-                Console.WriteLine("Error");
-                return Validation(out a, out b, out c);
+                Console.WriteLine("Error. Try again");
+                return Validation(out a);
             }
             return true;
-
         }
-        static bool Exist(double a, double b, double c)
+        static void Main(string[] args)
         {
-            if (a + b > c && a+c>b && b+c>a)
-                return true;
-            else
+            int a, k;
+            Console.WriteLine("Ввведите размер массива");
+            Validation(out a);
+            Random r = new Random();

```

Comment:
```suggestion
            Random random = new Random();

```
Т.к. переменные (и не только) нужно называть говорящими и понятными названиями, по которым станет понятно, что они означают.